src/creature.cpp

```diff
@@ -614,6 +614,24 @@ void Creature::onCreatureMove(Creature* creature, const Tile* newTile, const Pos
 	}
 }
 
+std::vector<Creature*> Creature::getKillers()
+{
+	std::vector<Creature*> killers;
```

Comment:
killers.reserve(damageMap.size())?

Reply:
I already did it, then I push the changes, I am not sure if this improves efficiency, but in theory it should, I say it because anyway, the reserve will not fill completely in all cases, but if it is ok have a limit and save some memory